### ISSUE TITLE
1.10 jar file is broken on Linux

### ISSUE THREAD

**Issue by xpander69** [Idrinth/WARAddonClient#137]
```
Exception in thread "main" java.lang.UnsupportedOperationException: Modifier not supported
	at sun.nio.fs.LinuxWatchService$Poller.implRegister(LinuxWatchService.java:237)
	at sun.nio.fs.AbstractPoller.processRequests(AbstractPoller.java:260)
	at sun.nio.fs.LinuxWatchService$Poller.run(LinuxWatchService.java:364)
	at java.lang.Thread.run(Thread.java:748)
```
edit:

And running the standalone version through wine gives this:

```
Exception in thread "main" java.lang.ExceptionInInitializerError
        at java.base/javax.crypto.Cipher.getInstance(Cipher.java:548)
        at java.base/sun.security.ssl.SSLCipher.isTransformationAvailable(SSLCipher.java:510)
        at java.base/sun.security.ssl.SSLCipher.<init>(SSLCipher.java:499)
        at java.base/sun.security.ssl.SSLCipher.<clinit>(SSLCipher.java:82)
        at java.base/sun.security.ssl.CipherSuite.<clinit>(CipherSuite.java:69)
        at java.base/sun.security.ssl.SSLContextImpl.getApplicableSupportedCipherSuites(SSLContextIm
pl.java:343)
        at java.base/sun.security.ssl.SSLContextImpl$AbstractTLSContext.<clinit>(SSLContextImpl.java
:556)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:340)
        at java.base/java.security.Provider$Service.getImplClass(Provider.java:1844)
        at java.base/java.security.Provider$Service.newInstance(Provider.java:1820)
        at java.base/sun.security.jca.GetInstance.getInstance(GetInstance.java:236)
        at java.base/sun.security.jca.GetInstance.getInstance(GetInstance.java:164)
        at java.base/javax.net.ssl.SSLContext.getInstance(SSLContext.java:184)
        at de.idrinth.waraddonclient.model.TrustManager.getStore(TrustManager.java:53)
        at de.idrinth.waraddonclient.model.TrustManager.<init>(TrustManager.java:22)
        at de.idrinth.waraddonclient.Main.main(Main.java:41)
Caused by: java.lang.SecurityException: Can not initialize cryptographic mechanism
        at java.base/javax.crypto.JceSecurity.<clinit>(JceSecurity.java:121)
        ... 17 more
Caused by: java.lang.SecurityException: Can't read cryptographic policy directory: unlimited
        at java.base/javax.crypto.JceSecurity.setupJurisdictionPolicies(JceSecurity.java:335)
        at java.base/javax.crypto.JceSecurity$1.run(JceSecurity.java:112)
        at java.base/javax.crypto.JceSecurity$1.run(JceSecurity.java:109)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:554)
        at java.base/javax.crypto.JceSecurity.<clinit>(JceSecurity.java:108)
        ... 17 more
0100:fixme:kernelbase:AppPolicyGetProcessTerminationMethod FFFFFFFFFFFFFFFA, 000000000022FDB8

```

**Comment by Idrinth** [comment]
should be fixed with 1.10.1

**Comment by xpander69** [comment]
Works. Thank you!
Also GTK+ theme is option now, so it matches my system dark theme... Superb job!
![ss_11052021_20 59 40](https://user-images.githubusercontent.com/1345116/117862956-e7966400-b29b-11eb-89e1-332d50976257.png)
